hudi-cli/src/main/java/org/apache/hudi/cli/HoodieCLI.java

```diff
@@ -85,7 +86,9 @@ public static void initFS(boolean force) throws IOException {
   }
 
   public static void refreshTableMetadata() {
-    setTableMetaClient(HoodieTableMetaClient.builder().setConf(HoodieCLI.conf).setBasePath(basePath).setLoadActiveTimelineOnLoad(false).setConsistencyGuardConfig(HoodieCLI.consistencyGuardConfig)
+    setTableMetaClient(HoodieTableMetaClient.builder().setConf(HoodieCLI.conf).setBasePath(basePath).setLoadActiveTimelineOnLoad(false)
+        .setConsistencyGuardConfig(HoodieCLI.consistencyGuardConfig)
+        .setTimeGeneratorConfig(HoodieTimeGeneratorConfig.defaultConfig(basePath))
         .setLayoutVersion(Option.of(layoutVersion)).build());
```

Comment:
Can we use the default config here, I saw some `createNewInstantTime` callings from some of the CLI commands.

Reply:
Added [HUDI-6884](https://issues.apache.org/jira/browse/HUDI-6884) to track this follow-up.